VersionControlUI/src/DiffManager.h

```diff
@@ -181,23 +204,30 @@ class VERSIONCONTROLUI_API DiffManager
 		 * Check if change associated with id should be visualized (e.g. if the change is part of a name change, does it
 		 * satisfy the visualization constraint).
 		 */
-		bool shouldShowChange(Model::NodeIdType id);
+		static bool shouldShowChange(Model::NodeIdType id);
 
 		QString project_;
 		QList<Model::SymbolMatcher> contextUnitMatcherPriorityList_;
 
 		// if set specifies which node we are interested in, used for history
 		Model::NodeIdType targetNodeId_;
 
-		QHash<QString, QPair<QString, Model::NodeIdType>> nameChanges_;
-		QHash<Model::NodeIdType, bool> nameChangesIdsIsNameText_;
-		NameChangeVisualizations nameChangeVisualization_{Summary};
+		QHash<QString, QPair<QString, Model::NodeIdType>> nameChangeInformation_;
+		static QHash<Model::NodeIdType, bool> nameChangesIdsIsNameText_;
+		static QList<ChangeWithNodes> nameChanges_;
+		static NameChangeVisualizations nameChangeVisualization_;
+		static QList<int> nameChangeOnZoomHandlerIds_;
+		static QSet<Visualization::Item*> nameChangesScaledByAncestor_;
```

Comment:
I did it because the command `toggleNameChangeHighlights` is static. They contain the information of the last diff and I use this to toggle the highlights.